sys/include/net/gnrc.h

```diff
@@ -144,9 +144,10 @@
  * }
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * @note It is necessary to react with a reply message to the other thread for
- *       @ref GNRC_NETAPI_MSG_TYPE_SET and @ref GNRC_NETAPI_MSG_TYPE_GET
- *       commands
+ * @note When receiving a message of type @ref GNRC_NETAPI_MSG_TYPE_SET or
+ *       @ref GNRC_NETAPI_MSG_TYPE_GET, it is necessary to acknowledge it by
+ *       calling @ref msg_reply() with a message of type
+ *       @ref GNRC_NETAPI_MSG_TYPE_ACK.
```

Comment:
Oh, that makes a lot of sense, thanks! I've changed it to
"[...] with a message of type @ref GNRC_NETAPI_MSG_TYPE_ACK which contains the actual size of the GET message's content on success or an error code otherwise."

now... But I'm wondering, this is quite a tricky thing to explain & get right, where would be the correct place to mention this in detail?